app/models/user.rb

```diff
@@ -21,5 +21,4 @@ class User < ApplicationRecord
   validates :summary, length: { maximum: 500 }
   validates :status, length: { maximum: 100 }
 
-  has_many_attached :photos
 end
```

Comment:
Why are you deleting above line? I couldn't get photo display to work on UI without it, I think it is needed.

Reply:
In the current version there is a has_many_attached :photos already since the weekend :  See below picture from git line 14:

![image](https://user-images.githubusercontent.com/21210868/126268827-69bf8376-4912-4c5d-a51e-b4e3abf121a8.png)